docs/toc.md

```diff
@@ -7,6 +7,7 @@
 ### [REST client](csharp/tutorials/console-webapiclient.md)
 ### [Working with LINQ](csharp/tutorials/working-with-linq.md)
 ### [Microservices hosted in Docker](csharp/tutorials/microservices.md)
+### [Building Docker Images for .NET Core Applications](docker/building-net-docker-images.md)
```

Comment:
I think I would put this under the .NET Core section, not in the C# section. 

Thoughts? @mairaw @richlander @cartermp